packages/header-app/src/components/openhim.appbar.component.tsx

```diff
@@ -29,24 +29,24 @@ const navbarTextStyle = {
   fontSize: '14px',
   letterSpacing: '0.4px',
   lineHeight: '24px',
-  textTransform: 'uppercase',
+  textTransform: 'capitalize',
   fontWeight: '500',
   fontFamily: 'Roboto',
   color: alpha('#000', 0.6)
 }
 
 export default function OpenhimAppBar() {
   const [pages, setPages] = useState<Page[]>([
-    {name: 'HOME', link: '#!/portal'},
-    {name: 'DASHBOARD', link: '#!/dashboard'},
+    {name: 'Home', link: '#!/portal'},
+    {name: 'Dashboard', link: '#!/dashboard'},
     {
-      name: 'TRANSACTIONS',
+      name: 'Transactions',
       link: '#!/transactions',
       permissions: ['transaction-view-all']
     },
-    {name: 'CHANNELS', link: '#!/channels', permissions: ['channel-view-all']},
+    {name: 'Channels', link: '#!/channels', permissions: ['channel-view-all']},
     {
-      name: 'CLIENTS',
+      name: 'Clients',
```

Comment:
The main headings should be capitalized only the sub-items are the ones we need to fix as per design. 
![image](https://github.com/user-attachments/assets/ed648b59-aa09-410f-b200-c4254707816b)